Makefile

```diff
@@ -34,6 +40,13 @@ deploy-local-image:
 		$(eval tag=dev-$(shell git rev-parse HEAD)-$(shell date '+%Y%m%d%H%M%S'))
 		docker build -t $(repository):$(tag) .
 		docker push $(repository):$(tag)
-		cf7 target -o dfe-teacher-services -s teaching-vacancies-$(env)
-		cf7 push -f paas/web/manifest-docker-$(env).yml --var IMAGE_NAME=$(repository):$(tag)
-		cf7 push -f paas/worker/manifest-docker-$(env).yml --var IMAGE_NAME=$(repository):$(tag)
+		$(eval export TF_VAR_paas_sso_passcode=$(passcode))
+		$(eval export TF_WORKSPACE=$(env))
+		$(eval export TF_VAR_paas_app_docker_image=$(repository):$(tag))
+		terraform init -input=false terraform/app
+		terraform apply -input=false -var-file terraform/workspace-variables/$(env).tfvars -auto-approve terraform/app
```

Comment:
maybe remove `-auto-approve` for dev workflow?

Reply:
It's fairly safe to deploy to dev automatically and it doesn't slow down the dev workflow.